### ISSUE TITLE
Assets 324-326: input disease ID is obsolete

### ISSUE THREAD

**Issue by colleenXu** [NCATSTranslator/Tests#35]
Assets 324-326 (currently Case 47) use the disease ID `MONDO:0008345` (Idiopathic_pulmonary_fibrosis). 

However:
* [this ID is actually obsolete](https://monarchinitiative.org/MONDO:0008345). 
* [NodeNorm doesn't recognize this ID](https://nodenorm.ci.transltr.io/get_normalized_nodes?curie=MONDO%3A0008345)

So it makes sense that BTE has not been returning any results for this ID. 

Should we perhaps use a different ID? 

Note: [MyDisease](https://mydisease.info/v1/disease/MONDO:0008345) has mappings for this ID to other namespaces and reports that the new ID may be [MONDO:0800029](https://monarchinitiative.org/MONDO:0800029)

**Comment by colleenXu** [comment]
I'd like to raise the priority of this issue @maximusunc, because asset 324 is part of the sprint 4 test suite.

**Comment by isbluis** [comment]
As further evidence, this is the Query Graph that is being posted to ARAX.  Note the lack of node `id` for the Object Node `"ON"` ("Unknown"), likely due to NodeNorm not returning a valid disease:
```
{
  "edges": {
    "e01": {
      "attribute_constraints": [],
      "knowledge_type": "inferred",
      "object": "ON",
      "predicates": [
        "biolink:treats"
      ],
      "qualifier_constraints": [],
      "subject": "SN"
    }
  },
  "nodes": {
    "ON": {
      "categories": [
        "biolink:Disease"
      ],
      "constraints": [],
      "ids": [
        "Unknown"
      ],
      "is_set": false,
      "set_interpretation": "BATCH"
    },
    "SN": {
      "categories": [
        "biolink:ChemicalEntity"
      ],
      "constraints": [],
      "is_set": false,
      "set_interpretation": "BATCH"
    }
  }
}
```

No ARAs return any results for this test asset:
![image](https://github.com/NCATSTranslator/Tests/assets/37852256/2fe2c469-c9f2-47ee-9d5b-9db0437cd48f)

**Comment by gaurav** [comment]
NodeNorm doesn't have any obsolete identifiers in it for now (although we're thinking about it, see https://github.com/TranslatorSRI/Babel/issues/93). How about [MONDO:0800029 "interstitial lung disease 2"](https://monarchinitiative.org/MONDO:0800029)? It has "idiopathic pulmonary fibrosis" as a synonym, and this is what MONDO lists in its [`obsoletes.tsv` file](https://github.com/monarch-initiative/mondo/blob/4753018179c28f6a0bb6c35ab67a7623f65a4a5f/reports/obsoletes.tsv#L1722).

**Comment by isbluis** [comment]
Results from the latest Sprint 4 run for this Test look quite a bit better, FYI:
![image](https://github.com/NCATSTranslator/Tests/assets/37852256/5947b9f9-8972-4c1c-b9a2-1ef188f251a6)

**Comment by colleenXu** [comment]
Maybe this issue should be closed since the change has been merged?

**Comment by maximusunc** [comment]
Yep, this has been fixed in #89
